### ISSUE TITLE
Translation for "close" missing in Finnish, Norwegian and Swedish

### ISSUE THREAD

**Issue by jaller94** [KIProtect/klaro#137]
We're missing a translation for "close" as in "closing a window in the following language files:

* [ ] [Finnish](https://github.com/KIProtect/klaro/blob/master/src/translations/fi.yml)
* [ ] [Norwegian](https://github.com/KIProtect/klaro/blob/master/src/translations/no.yml)
* [ ] [Swedish](https://github.com/KIProtect/klaro/blob/master/src/translations/sv.yml)

**Comment by gitFrank** [comment]
Also is missing translation for "acceptSelected" in german.
Suggestion: acceptSelected:"Ausgewählte akzeptieren"

**Comment by adewes** [comment]
Yes, we have to find a better way to add missing translations I think.

**Comment by darylknight** [comment]
Finnish for closing a window is `sulje`

**Comment by Steddie** [comment]
Swedish word for close is

> Stäng

**Comment by ExtSol** [comment]
The keys and translations for all the related languages are missing:
close: Close
acceptAll: Accept all
acceptSelected: Accept selected

By the way, according to ISO 3166-1 the Alpha-2 code for Swedish is SE not SV - [see details
](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)
[Updated](https://github.com/ExtSol/klaro/tree/master/src/translations) some missing translations

**Comment by ExtSol** [comment]
> By the way, according to ISO 3166-1 the Alpha-2 code for Swedish is SE not SV - [see details](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)

I made a mistake, please accept my apologies.
The correct ISO code is: [639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
